lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/EmptyMultiRuntimeBootstrapConfigurationGeneratorImpl.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.reef.runtime.multi.client;
+
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.Tang;
+
+import javax.inject.Inject;
+
+/**
+ * Default implementation for MultiRuntimeBootstrapConfigurationGenerator that returns an empty configuration.
+ */
+final class EmptyMultiRuntimeBootstrapConfigurationGeneratorImpl
```

Comment:
For Local runtime (which is not supported but I use it for some testing/debugging) no additional configurations are required. I don't know what will be the case for Mesos or other environments but it may be the same. So prefer having an empty default then requiring developing an empty implementation each time it is not needed.